elide-spring/elide-spring-boot-autoconfigure/src/test/java/example/tests/ControllerTest.java

```diff
@@ -221,26 +231,499 @@ public void jsonForbiddenApiPatchTest() {
+                                                id("com.example.operations2"),
+                                                attributes(
+                                                        attr("commonName", "Foo2")
+                                                )
+                                        ))
+                                ),
+                                atomicOperation(AtomicOperationCode.update, "/group/com.example.operations2",
+                                        datum(resource(
+                                                type("group"),
+                                                id("com.example.operations2"),
+                                                attributes(
+                                                        attr("description", "Updated Description")
+                                                )
+                                        ))
+                                ),
+                                atomicOperation(AtomicOperationCode.add, "/group/com.example.operations2/products",
+                                        datum(resource(
+                                                type("product"),
+                                                id("com.example.operations.product1"),
+                                                attributes(
+                                                        attr("commonName", "Product1")
+                                                )
+                                        ))
+                                ),
+                                atomicOperation(AtomicOperationCode.update, "/group/com.example.operations2/products/com.example.operations.product1",
+                                        datum(resource(
+                                                type("product"),
+                                                id("com.example.operations.product1"),
+                                                attributes(
+                                                        attr("description", "Product1 Description")
+                                                )
+                                        ))
+                                )
+                        )
+                )
+                .when()
+                .post("/json/operations")
+                .then()
+                .statusCode(HttpStatus.SC_OK)
+                .extract();
+        Map<String, Object> groupAttributes = response.path("'atomic:results'[1].data.attributes");
+        assertEquals("Foo2", groupAttributes.get("commonName"));
+        assertEquals("Updated Description", groupAttributes.get("description"));
+
+        Map<String, Object> productAttributes = response.path("'atomic:results'[3].data.attributes");
+        assertEquals("Product1", productAttributes.get("commonName"));
+        assertEquals("Product1 Description", productAttributes.get("description"));
```

Comment:
There are a number of patch extenstion requests in ResourceIT (you can search for JSONAPI_CONTENT_TYPE_WITH_JSON_PATCH_EXTENSION).  They also verify that a subsequent GET request has the correct values.